packages/webpack-plugin/test/e2e/simple-production-project.spec.ts

```diff
@@ -17,8 +17,8 @@ describe(`(${project})`, () => {
         after
     );
 
-    it('renders css', async () => {
-        const source = await projectRunner.getBuildAsset('main.bundle.css');
+    it('renders css', () => {
+        const source = projectRunner.getBuildAsset('stylable.css');
```

Comment:
is it named stylable.css by default? or is it the config causing this?

Reply:
it's the default